mkdocs/docs/HPC/python.md

```diff
@@ -0,0 +1,164 @@
+
+## Loading packages
+
+If your python script does not use any third-party packages, you only have to load a Python module.
+If you need to use a third-party package, you will need to load the corresponding module.
+Loading such a module will also load a Python module, so you do not need to load both.
+
+For example, loading the `SciPy-bundle` module will also load the Python module:
+
+```bash
+$ module load SciPy-bundle/2023.11-gfbf-2023b
+$ module list
+Currently Loaded Modules:
+ ...
+ 21) Python/3.11.5-GCCcore-13.2.0
+ ...
+ 27) SciPy-bundle/2023.11-gfbf-2023b
+```
+
+## Example: Running Python scripts
+
+There are multiple ways to run Python scripts on the HPC systems. 
+You can start an interactive session on a cluster and run your script manually,
+or you can submit a job using a job script. 
+This section will show you how to run Python scripts using both methods on the following example script:
+
+```python title="script.py"
+import numpy as np
+
+random_numbers = np.random.rand(5)
+
+print(f"Random numbers: {random_numbers}")
+```
+
+First, we select a toolchain of the SciPy bundle module, which contains the `numpy` package, along with a Python module:
+
+```bash
+$ module avail SciPy-bundle
+```
+
+We will use the `SciPy-bundle/2023.11-gfbf-2023b` toolchain.
+Next, we decide on which cluster we want to run the script. 
+For this example, we will use the `donphan` cluster.
+
+To switch to the `donphan` cluster, we run:
+
+```bash
+$ module swap cluster/donphan
+```
+
+### Interactive session
+
+To run the script interactively, we need to start an interactive session on the cluster (`donphan` in this case):
+
+```bash
+$ qsub -I
+```
+
+Once the interactive session is started, we can load the `SciPy-bundle` module and run the script:
+
+```bash
+$ module load SciPy-bundle/2023.11-gfbf-2023b
+```
+
+Now we can run the script:
+
+```bash
+$ python script.py
+```
+
+### Job script
+
+To run the script using a job script, 
+we need to create a job script that will load the `SciPy-bundle` module and run the script:
+
+```bash title="jobscript.pbs"
+#!/bin/bash
+
+# Basic parameters
+#PBS -N python_job_example            ## Job name
+#PBS -l nodes=1:ppn=1                 ## 1 node, 1 processors per node
+#PBS -l walltime=01:00:00             ## Max time your job will run (no more than 72:00:00)
+
+module load SciPy-bundle/2023.11-gfbf-2023b   # Load the SciPy module, which includes the numpy package
+cd $PBS_O_WORKDIR                             # Change working directory to the location where the job was submitted
+python script.py                              # Run your Python script
+```
+
+To submit the job, run:
+
+```bash
+qsub jobscript.pbs
+```
+
+After some time, two files will be created in the directory where the job was submitted: 
+`python_job_example.o{{jobid}}` and `python_job_example.e{{jobid}}`, where {{jobid}} is the id of your job.
+The `.o` file contains the output of the job.
+
```

Comment:
Add small section/notes at the end to point to dedicated page on i) Python virtual environments ii) Jupyter notebooks